internal/nexodus/policy_linux.go

```diff
@@ -201,6 +200,19 @@ func (nx *Nexodus) nfPermitProtoPortAddrV4(chain string, rule public.ModelsSecur
 					return err
 				}
 			}
+		} else if rule.FromPort != 0 && rule.ToPort != 0 {
```

Comment:
what if one port is `0` and one isn't? That case is ignored. Does validation in the API catch that already? Even if so, it's probably worth logging an error here to indicate that something unexpected happened.

Reply:
Good catch, I changed the port validation in the API server so it would never reach the agent. ty.